lib/restivus.coffee

```diff
@@ -309,11 +311,20 @@ class @Restivus
       get: ->
         # Remove the given auth token from the user's account
         authToken = @request.headers['x-auth-token']
-        Meteor.users.update @user._id, {$pull: {'services.resume.loginTokens': {token: authToken}}}
+        hashedToken = Accounts._hashLoginToken authToken
+        tokenLocation = self.config.auth.token
+        index = tokenLocation.lastIndexOf '.'
+        tokenPath = tokenLocation.substring 0, index
+        tokenFieldName = tokenLocation.substring index + 1
```

Comment:
I like this way of doing it as well. Definitely more straightforward, as you suggested.